Pose pickPos
Pose placePos
---
bool isPickFinshed
